unity-renderer/Assets/Rendering/Culling/CullingController.cs

```diff
@@ -91,6 +96,17 @@ public void Start()
             StartInternal();
         }
 
+        /// <summary>
+        /// Starts culling update coroutine, if it was already running it restarts.
+        /// The coroutine will keep running until Stop() is called or this class is disposed.
+        /// </summary>
+        public void Restart()
```

Comment:
this new added method is not being called?

Reply:
This will be called from the loading system to stop calculating GameObjects that belong to a different position (And will not be needed anymore), right now is not being called, I am just adding the capability of doing so as requested in the new loading screen